addons/utm/wizard/utm_merge_utm_campaign.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class MergeUtmCampaign(models.TransientModel):
+    """
+        Merger UTM campaigns together.
+    """
+
+    _name = 'utm.merge.utm.campaign'
```

Comment:
Guidelines: `_name = 'utm.campaign.merge'`

Also rename file to `utm_campaign_merge.py`